docs/fsharp/tutorials/getting-started/getting-started-visual-studio.md

```diff
@@ -5,168 +5,147 @@ keywords: visual f#, f#, functional programming
 author: dend
 manager: danielfe
 ms.date: 05/16/2016
-ms.topic: language-reference
+ms.topic: article
 ms.prod: visual-studio-dev14
 ms.technology: devlang-fsharp
 ms.assetid: 8db75596-19a9-4eda-b20d-a12d517c8cc1 
 ---
 
 # Getting Started with F# in Visual Studio
 
-> [!NOTE]
-This article is not up to date with the latest Visual Studio.  It will be updated.
+F# and the Visual F# tooling are supported in the Visual Studio IDE.  To begin, you should [download Visual Studio](https://visualstudio.com/downloads/).  The rest of the article will assume Visual Studio 2015 Community Edition.
```

Comment:
Instead of saying "The rest of the article will assume Visual Studio 2015 Community Edition.", wouldn't be better to say something along the lines of "This article uses the Visual Studio 2015 Community Edition, but you can use F# with the version of your choice."?

To begin, you should [download Visual Studio](https://visualstudio.com/downloads/). -> To begin, you should [download Visual Studio](https://visualstudio.com/downloads/), if you haven't already.